### PR TITLE
Bugfix/search_responsive_layout

### PR DESCRIPTION
**Pull request by RuslanHrybchuk**

### CODE REVIEW

File: src/app/component/core/search-popup/search-popup.component.html [hunk 1]
```diff
@@ -11,47 +11,49 @@
     </div>
   </div>
```
Comment (nabermi2020) [inline src/app/component/core/search-popup/search-popup.component.html:12]: image

Comment (nabermi2020) [reply src/app/component/core/search-popup/search-popup.component.html:12]: Search results area should has width 100%, and results should be aligned to the center

Comment (nabermi2020) [reply src/app/component/core/search-popup/search-popup.component.html:12]: image

Also, please, open eco-news tab -> scroll down  -> open search -> enter something

Comment (nabermi2020) [reply src/app/component/core/search-popup/search-popup.component.html:12]: You don't see anything, 'cause block with class '.search-content-wrapper' is still on the top, as a result you should use position fixed for '.search-content-wrapper'

Comment (RuslanHrybchuk) [reply src/app/component/core/search-popup/search-popup.component.html:12]: fixed

**Pull request merged**